### ISSUE TITLE
Manifest merger problem

### ISSUE THREAD

**Issue by DanChaltiel** [juliome10/DoubleViewPager#3]
I have a problem when I import the library (I use gradle) : your manifest allow backup and mine doesn't.

Of course I can change my value to true, but for now I would rather not to.

(and oddly, tools:replace and tool:node don't work)

This is the stack :

```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:9:9-36
    is also present at [com.github.juliome10:DoubleViewPager:1.0.1] AndroidManifest.xml:12:9-35 value=(true).
    Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:7:5-47:19 to override.
```

**Comment by juliome10** [comment]
Removed unused manifest attributes on v.1.0.3.

**Comment by DanChaltiel** [comment]
Are you sure ?
It still doesn't work with `compile 'com.github.juliome10:DoubleViewPager:1.0.3@aar'`

**Comment by juliome10** [comment]
@DanChaltiel Sorry! fixed right now on v1.0.4 ;)
